source/form.html

```diff
@@ -38,45 +38,42 @@ <h1 class="page-main__title">Подбор программы</h1>
             required></label>
-        <label class="personal-info__cat-weight">Вес (кг):*<input type="number" placeholder="7" required></label>
-        <label class="personal-info__cat-age">Возраст (лет):<input type="number" placeholder="7"></label>
-        <fieldset class="form__radio-buttons radio-buttons">
-          <label class="radio-buttons__label">Похудение<input class="radio-buttons__item radio-buttons__item--active"
-              type="radio" name="program-goal" value="Похудение" checked></label>
-          <label class="radio-buttons__label">Набор массы<input class="radio-buttons__item" type="radio"
-              name="program-goal" value="Набор массы"></label>
-          <label class="radio-buttons__label">Не знаю (Нужен ваш совет)<input class="radio-buttons__item" type="radio"
-              name="program-goal" value="Не знаю (Нужен ваш совет)"></label>
-        </fieldset>
+        <label class="form__label">Вес (кг):*<input class="form__input" type="number" placeholder="7" required></label>
+        <label class="form__label">Возраст (лет):<input class="form__input" type="number" placeholder="7"></label>
+        <label class="form__label radio-label">Похудение<input class="form__input form__input--active radio-button"
+            type="radio" name="program-goal" value="Похудение" checked></label>
+        <label class="form__label radio-label">Набор массы<input class="form__input radio-button" type="radio"
+            name="program-goal" value="Набор массы"></label>
+        <label class="form__label radio-label">Не знаю (Нужен ваш совет)<input class="form__input radio-button"
+            type="radio" name="program-goal" value="Не знаю (Нужен ваш совет)"></label>
       </div>
 
-      <div class="form__contacts contacts">
-        <h2 class="contacts__title form-subtitle">Контактные данные
+      <div class="form__contacts">
+        <h2 class="form__subtitle">Контактные данные
           (владельца кота)</h2>
-        <label class="contacts__email">E-mail:*<input type="email" placeholder="kuklachev@gmail.com" required></label>
-        <label class="contacts__phone">Телефон:*<input type="tel" placeholder="8 (960) 900-60-90" required></label>
+        <label class="form__label">E-mail:*<input class="form__input" type="email" placeholder="kuklachev@gmail.com"
+            required></label>
+        <label class="form__label">Телефон:*<input class="form__input" type="tel" placeholder="8 (960) 900-60-90"
+            required></label>
       </div>
 
-      <div class="form__comments comments">
-        <h2 class="comments__title form-subtitle">Комментарий</h2>
-        <label class="visually-hidden">Повадки кота.<textarea class="comments__textarea"
+      <div class="form__comments">
+        <h2 class="form__subtitle">Комментарий</h2>
+        <label class="visually-hidden">Повадки кота.<textarea class="form__input textarea"
```

Comment:
```suggestion
        <label class="visually-hidden">Повадки кота.<textarea class="form__input form__input--textarea"
```

textarea - это мод, тк она как инпут, просто в высоту больше